website/docs/r/bigquery_dataset.html.markdown

```diff
@@ -26,6 +26,17 @@ resource "google_bigquery_dataset" "default" {
   labels {
     env = "default"
   }
+
+  access = [
```

Comment:
this should be:
```
access {
  role   = "READER"
  domain = "example.com"
}
access {
  role           = "WRITER"
  group_by_email = "writers@example.com"
}
```

There's nothing wrong with the other way, this is just more canonical.

Reply:
Done.